src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -337,6 +340,53 @@ to sanitize his mailbox denormalization.
 In order to ensure being offline, stop the traffic on SMTP, JMAP and IMAP ports, for example via re-configuration or 
 firewall rules.
 
+### Mailboxes Counters
+
+Cassandra maintains a per mailbox projection for message count and unseen message count. Failures during the denormalization 
+process will lead to inconsistencies between the counts and the content of `imapUidTable`.
```

Comment:
there's a lack of symmetry: either you talk only of tables or only of concepts but you should mix the two